app/src/renderer/components/network/PageBlock.vue

```diff
@@ -94,8 +94,15 @@ export default {
     this.getBlock()
   },
   methods: {
-    async getBlock() {
-      await this.$store.dispatch(`queryBlockInfo`, this.$route.params.height)
+    async getBlock({ $store, $route, $router } = this) {
+      const block = await $store.dispatch(
+        `queryBlockInfo`,
+        $route.params.height
+      )
+
+      if (!block) {
```

Comment:
should we include something that makes sure the block number is greater than the current block? not receiving a block from this request could be other issues aside from 404, right?